coral/src/app/layout/header/HeaderNavigation.test.tsx

```diff
@@ -1,13 +1,20 @@
 import { cleanup, screen, within } from "@testing-library/react";
 import HeaderNavigation from "src/app/layout/header/HeaderNavigation";
+import { Routes } from "src/app/router_utils";
 import { customRender } from "src/services/test-utils/render-with-wrappers";
 import {
   tabThroughBackward,
   tabThroughForward,
 } from "src/services/test-utils/tabbing";
 
+const isFeatureFlagActiveMock = jest.fn();
+
+jest.mock("src/services/feature-flags/utils", () => ({
```

Comment:
I think it could be a good idea to actually add the feature flag that we need to be active 🤔  Not because it's important for the test, but when we remove the flag, it will come up when searching for `FEATURE_FLAG_TOPNAV_DROPDOWN` and we'll know that we can safely remove the mock (or part of it, in case there are multipe flags mocked) 

Not important for this PR now, but if you agree, I would later do a small follow up on the docs to describe it.

Reply:
Yes that makes sense :o